u19_pipeline/recording.py

```diff
@@ -0,0 +1,181 @@
+import datajoint as dj
+from u19_pipeline import lab, task, subject
+
+schema = dj.schema('u19_recording')
+
+from u19_pipeline import acquisition, subject
+
+
+@schema
+class RecordingModality(dj.Lookup):
+     definition = """
+     recording_modality:   varchar(64)                  # modalities for recording (ephys, imaging, video_recording, etc.) 
+     ---
+     modality_description: varchar(255)                 # description for the modality
+     root_direcory:        varchar(255)                 # root directory where that modality is stored (e.g. ephys = /braininit/Data/eletrophysiology)
```

Comment:
```suggestion
     root_directory:        varchar(255)                 # root directory where that modality is stored
```
Since we have the `contents` property, perhaps we don't need the example.

Reply:
Agreed !